### PR TITLE
Fire specific event for non-0 exit status command

### PR DESCRIPTION
**Pull request by mikaelrandy**
With this PR, a command who finish with a non-0 exit status will lead to fire a specific event in StatsBundle : `m6web.command.error`, in addition to `m6web.command.terminate` event who is ever fired at command exit.

There is 2 limits with this PR : 
* the command's developper has to explicitly return non-0 exit status to consider the command's execution as failed
* a fatal error in the command do not lead to any event's fire.

### CODE REVIEW

File: src/Listener/ConsoleListener.php [hunk 1]
```diff
@@ -46,10 +47,15 @@ public function onCommand(BaseConsoleEvent $e)
     }
 
     /**
-     * @param BaseConsoleEvent $e
+     * @param ConsoleTerminateEvent $e
      */
-    public function onTerminate(BaseConsoleEvent $e)
+    public function onTerminate(ConsoleTerminateEvent $e)
```
Comment (mikaelrandy) [inline src/Listener/ConsoleListener.php]: Here is a possible compatibility break, but as Symfony always give this Event class in all Symfony 2.3+ versions, i have considered as a non-BC break update.

File: composer.json [hunk 2]
```diff
@@ -14,7 +14,6 @@
     "config": {
         "vendor-dir": "vendor"
     },
-    "minimum-stability" : "dev",
```
Comment (omansour) [inline composer.json]: :+1:

**Comment by jdlabails** [comment]
:+1:

**Comment by omansour** [comment]
:+1:

**Comment by mguillermin** [comment]
It was too late to comment about the missing documentation, so I've made a PR for that -> #38

**Comment by mikaelrandy** [comment]
v2.12.0

**Pull request merged**